InnoClinic.AuthorizationAPI/BLL/Models/Validators/LoginDTOValidator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using BLL.Models.DTOs;
+using FluentValidation;
+
+namespace BLL.Models.Validators;
+
+public class LoginDTOValidator : AbstractValidator<LogInDTO>
+{
+    public LoginDTOValidator()
+    {
+        RuleFor(x => x.Email)
+            .NotEmpty().WithMessage("Please, enter the email")
+            .EmailAddress().WithMessage("You've entered an invalid email");
+
+        RuleFor(x => x.Password)
+            .NotEmpty().WithMessage("Please, enter the password")
+            .Length(6, 15).WithMessage("Password must be 6-15 characters")
+            .Matches(@"^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[^a-zA-Z0-9]).+$")
+            .WithMessage("Password must contain at least one uppercase, lowercase, digit and special character");
```

Comment:
I think you don't need this check as password has already exist. We aren't going to create the new one, we're only checking if it is correct